composer.json

```diff
@@ -67,8 +67,14 @@
     "prefer-stable": true,
     "scripts": {
         "analyze": "psalm",
-        "check-style": "phpcs -p --standard=PSR12 config/ resources/ src/ tests/ '--ignore=*__snapshots_*'",
-        "fix-style": "phpcbf -p --standard=PSR12 config/ resources/ src/ tests/ '--ignore=*__snapshots_*'",
+        "check-style": [
+            "php-cs-fixer fix --diff --diff-format=udiff --dry-run",
+            "php-cs-fixer fix --diff --diff-format=udiff --dry-run --config=.php_cs.tests.php"
+        ],
+        "fix-style": [
```

Comment:
Primary composer commands unchanged, so transparently will work for those used to them